### PR TITLE
Fix docstring for Series.count()

### PR DESCRIPTION
**Pull request by akharche**

### CODE REVIEW

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 1]
```diff
@@ -4448,7 +4448,7 @@ def hpat_pandas_series_count(self, level=None):
 
     Limitations
     -----------
-    - Parameter level is currently unsupported by Intel Scalable Dataframe Compiler
+    Parameter ``level`` is supported only with default value ``None``.
```
Comment (PokhodenkoSA) [inline sdc/datatypes/hpat_pandas_series_functions.py]: Is it new correct way to define unsupported parameters?

Comment (densmirn) [reply sdc/datatypes/hpat_pandas_series_functions.py]: I'm following the point `shouldn't be phrase unsupported in SDC` in the checklist.

Comment (densmirn) [reply sdc/datatypes/hpat_pandas_series_functions.py]: Moreover we cannot say that a parameter is unsupported at all, at the least the default value is valid

Comment (PokhodenkoSA) [reply sdc/datatypes/hpat_pandas_series_functions.py]: This point in checklist was about ``Unsupported by Intel SDC.`` which appears when there is no corresponding documentation in SDC for method from Pandas.

Comment (akharche) [reply sdc/datatypes/hpat_pandas_series_functions.py]: Fixed

**Pull request merged**